.github/workflows/release-micromamba.yml

```diff
@@ -0,0 +1,194 @@
+name: release-micromamba
+
+on:
+  workflow_dispatch: null
+  pull_request: null
+  push: null
+
+jobs:
+  release-micromamba-linux-osx:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: look for a tag
+        run: |
+          echo "found tag: '$(git tag --points-at HEAD)'"
+          echo "GIT_TAG=$(git tag --points-at HEAD)" >> $GITHUB_ENV
+
+      - name: free disk space
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          sudo swapoff -a
+          sudo rm -f /swapfile
+          sudo apt clean
+          docker rmi $(docker image ls -aq)
+          df -h
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          python-version: 3.8
+          channels: conda-forge,defaults
+          channel-priority: strict
+
+      - name: install deps
+        shell: bash -l {0}
+        run: |
+          conda env create -q --file environment-unix.yml
+          conda activate mamba-tests
+          conda list
+
+      - name: build micromamba
+        shell: bash -l {0}
+        run: |
+          conda activate mamba-tests
+          mkdir -p ~/.conda
+          touch ~/.conda/environments.txt
+
+          curl -Ls https://github.com/CLIUtils/CLI11/releases/download/v1.9.1/CLI11.hpp -o include/mamba/CLI.hpp
```

Comment:
Can we use this `CLI11.hpp` file for win and unix? https://github.com/conda-forge/micromamba-feedstock/blob/master/recipe/CLI11.hpp

This is a later master version of CLI11, and the (important) difference is that one can put multiple `-c` arguments anywhere on the command line. The latest release can only handle `-c` arguments properly when they come at the end of the CLI arguments.

I asked the CLI team for a release but they are quite busy. cc @henryiii